Userland/Applications/HexEditor/main.cpp

```diff
@@ -71,15 +58,16 @@ int main(int argc, char** argv)
     window->show();
     window->set_icon(app_icon.bitmap_for_size(16));
 
-    if (!file_to_edit.is_empty()) {
-        auto file = Core::File::open(file_to_edit, Core::OpenMode::ReadOnly);
+    if (argc > 1) {
+        auto response = FileSystemAccessClient::Client::the().request_file_read_only_approved(window->window_id(), argv[1]);
 
-        if (file.is_error()) {
-            GUI::MessageBox::show_error(window, String::formatted("Opening \"{}\" failed: {}", file_to_edit, file.error()));
+        if (response.error != 0) {
+            if (response.error != -1)
```

Comment:
These numbers
What do they meaaaaan

Reply:
(please less magic number and more enum, kthx)

Reply:
I'm not really sure what these error codes represent, I've just been following the template that was originall there in the applications that used FSAS

Reply:
Alright, if you think it's better in another PR, sure.

Reply:
It seems that `FileSystemAccessServer` returns `errno` in some cases for the return value, so I don't know if there's any nice way of using an enum (unless we have one for all of the `errno` errors?

Reply:
if it's just errno, please use the appropriate names instead of numeric literals.
not a blocker for this PR though.